_posts/en/posts/2016-10-28-segwit-costs.md

```diff
@@ -298,8 +298,8 @@ Either approach to a hard-fork would make it possible to simultaneously drastica
 
 Many of the benefits of segwit could logically be separated into independent changes, and evaluated and deployed separately. The implementation requirements for the various features are, however, closely related:
 
- * Linear scaling of sighash operations: the CHECKSIG and CHECKMULTISIG opcodes need replacement.
- * Signing of input values: the CHECKSIG and CHECKMULTISIG opcodes need replacement.
+ * Quadratic scaling of sighash operations: the CHECKSIG and CHECKMULTISIG opcodes need replacement.
+ * Non-signing of input values: the CHECKSIG and CHECKMULTISIG opcodes need replacement.
```

Comment:
I don't think these should be changed.  The text before the colon describes the segwit feature being implemented (linear sig hashing, signing input values, 128-bit collision resistance for multiparty multisig, and more script upgrade paths); the text after the colon describes the change that needs to be made to the protocol.

Reply:
Good point, updated it